spanner/api/Program.cs

```diff
@@ -0,0 +1,989 @@
+            {
+                // Only startTitle provided, show message that user must 
+                // provide both startTitle and endTitle or exclude them 
+                // from their command.
+                Console.WriteLine("Please provide values for both the start "
+                    + "of the title index and the end of the title index.");
+                Console.WriteLine("Or you can exclude both to run the "
+                    + "query with default values.");
+                return 0;
+            }
+            else if (!string.IsNullOrEmpty(startTitle)
+                && !string.IsNullOrEmpty(endTitle))
+            {
+                // Both startTitle and endTitle provided, include them both in
+                // the method call to QueryDataWithIndexAsync.
+                response = Task.Run(async () => await QueryDataWithIndexAsync(
+                        projectId, instanceId, databaseId,
+                        startTitle, endTitle));
+            }
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Operation status: {response.Status}");
+            return 0;
+        }
+
+        public static object CreateDatabase(string projectId,
+            string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () => await CreateDatabaseAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            Console.WriteLine($"Created database {databaseId} on instance "
+                + $"{instanceId}");
+            return 0;
+        }
+
+        public static object AddIndex(string projectId,
+    string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await AddIndexAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+        public static object AddStoringIndex(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await AddStoringIndexAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object QueryDataWithStoringIndex(string projectId,
+            string instanceId, string databaseId, string startTitle, string endTitle)
+        {
+            var response = new Task(() => { });

```

Comment:
Sticking with older approach for creating Tasks. TODO: revisit this in a future update.